--- a/pdns/Makefile.am
+++ b/pdns/Makefile.am
@@ -81,7 +81,6 @@ CLEANFILES = \
 	backends/gsql/gsqlbackend.gcda \
 	backends/gsql/gsqlbackend.gcno \
 	backends/gsql/gsqlbackend.gcov \
-	pdns.conf-dist \
 	apidocfiles.h
 
 if !HAVE_API_OPENAPI_JSON
@@ -115,8 +114,6 @@ apidocfiles.h: api-openapi.yaml api-open
 	$(AM_V_GEN)$(srcdir)/incfiles $^ > $@.tmp
 	@mv $@.tmp $@
 
-sysconf_DATA = pdns.conf-dist
-
 sbin_PROGRAMS = pdns_server
 bin_PROGRAMS = \
 	pdns_control \
@@ -154,7 +151,7 @@ if IXFRDIST
 bin_PROGRAMS += \
 	ixfrdist
 
-sysconf_DATA += \
+sysconf_DATA = \
 	ixfrdist.example.yml
 endif
 
@@ -1378,9 +1375,6 @@ dnspcap2protobuf_LDADD = \
 	$(BOOST_PROGRAM_OPTIONS_LIBS) \
 	$(RT_LIBS)
 
-pdns.conf-dist: pdns_server
-	$(AM_V_GEN)./pdns_server --config=default > $@
-
 testrunner_SOURCES = \
 	arguments.cc \
 	auth-caches.cc auth-caches.hh \
