--- a/pdns/Makefile.am
+++ b/pdns/Makefile.am
@@ -78,7 +78,6 @@ CLEANFILES = \
 	backends/gsql/gsqlbackend.gcda \
 	backends/gsql/gsqlbackend.gcno \
 	backends/gsql/gsqlbackend.gcov \
-	pdns.conf-dist \
 	apidocfiles.h
 
 if !HAVE_API_SWAGGER_JSON
@@ -112,8 +111,6 @@ apidocfiles.h: api-swagger.yaml api-swag
 	$(AM_V_GEN)$(srcdir)/incfiles $^ > $@.tmp
 	@mv $@.tmp $@
 
-sysconf_DATA = pdns.conf-dist
-
 sbin_PROGRAMS = pdns_server
 bin_PROGRAMS = \
 	pdns_control \
@@ -151,7 +148,7 @@ if IXFRDIST
 bin_PROGRAMS += \
 	ixfrdist
 
-sysconf_DATA += \
+sysconf_DATA = \
 	ixfrdist.example.yml
 endif
 
@@ -1333,9 +1330,6 @@ dnspcap2protobuf_LDADD = \
 	$(BOOST_PROGRAM_OPTIONS_LIBS) \
 	$(RT_LIBS)
 
-pdns.conf-dist: pdns_server
-	$(AM_V_GEN)./pdns_server --config=default > $@
-
 testrunner_SOURCES = \
 	arguments.cc \
 	auth-caches.cc auth-caches.hh \
